code/modules/clothing/suits/f13armor.dm

```diff
@@ -554,7 +554,7 @@
 
 /obj/item/clothing/suit/armor/f13/power_armor/run_block(mob/living/owner, atom/object, damage, attack_text, attack_type, armour_penetration, mob/attacker, def_zone, final_block_chance, list/block_return)
 	if((attack_type == ATTACK_TYPE_PROJECTILE) && (def_zone in protected_zones))
-		if(prob(70) && (damage < deflect_damage)) // Weak projectiles like shrapnel get deflected
+		if(prob(70) && (damage < deflect_damage) && (armour_penetration <= 0)) // Weak projectiles like shrapnel get deflected
```

Comment:
I'd honestly make it so the chance itself scaled with projectile damage and AP, but it works too.

Reply:
I'd rather honestly just keep it like this to prevent stupid deflections like rockets ceasing to exist.